src/components/viewer.js

```diff
@@ -20,10 +20,26 @@ const renderChildren = (nodes, isListItem) =>
     }
     // Text node
     if (typeof node === 'string') {
-      return node;
+      const line = node.replace(/\n/g, "");
```

Comment:
ah, turns out internally we're wrapping the lines in a `<p>` tag, which doesn't allow for nesting of `block` displayed elements. Theres other issues with the creation of the `<Tag>` component will account for not just text, but plotly charts and images as well. Beyond that, just wrapping the text in a `pre` tag is not preserving the line breaks for me, even when I remove any styling that come along with it. The current solution seems to be the easiest at the moment.